controller/setting_controller.go

```diff
@@ -1858,6 +1863,7 @@ func (info *ClusterInfo) collectVolumesInfo() error {
 	info.structFields.fields.AppendCounted(restoreVolumeRecurringJobCountStruct)
 	info.structFields.fields.AppendCounted(snapshotDataIntegrityCountStruct)
 	info.structFields.fields.AppendCounted(unmapMarkSnapChainRemovedCountStruct)
+	info.structFields.fields.AppendCounted(freezeFilesystemForSnapshotCountStruct)
```

Comment:
We also need to update the upgrade responder server as well https://github.com/longhorn/longhorn/blob/master/deploy/upgrade_responder_server/chart-values.yaml

Reply:
Got it. Added https://github.com/longhorn/longhorn/pull/8447/commits/2b7f9282015a693ae63651cfa9233444b26e8a7b.